lib/fake_braintree/sinatra_app.rb

```diff
@@ -7,12 +7,16 @@
 require 'fake_braintree/address'
 require 'fake_braintree/payment_method'
 require 'fake_braintree/transaction'
+require 'fake_braintree/client_token'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.